__snapshots__/index.test.js.snap

```diff
@@ -214,6 +216,65 @@ export function Thing(props) {
 }"
 `;
 
+exports[`babel-preset-zapier transpiles TSX with type keyword 1`] = `""`;
```

Comment:
This is expected - imports/exports of `type`s should be stripped out completely.